src/components.d.ts

```diff
@@ -45,6 +45,10 @@ export namespace Components {
         "variant"?: AlertHeaderVariannt;
     }
     interface BdsBanner {
+        /**
+          * Specifies the background color to use. The default background color is $color-primary-main.
+         */
+        "background": string;
```

Comment:
Wouldn't it be interesting to just use an external class to customize the background? Or we can start using CSS Variables

Reply:
what do you suggest? to customize outside the component by a CSS class?

Reply:
yes! :)